presto-main/src/main/java/com/facebook/presto/operator/scalar/JoniRegexpFunctions.java

```diff
@@ -307,6 +309,16 @@ public static Block regexpSplit(@SqlType("varchar(x)") Slice source, @SqlType(Jo
         return blockBuilder.build();
     }
 
+    private static int getMatchingOffset(Matcher matcher, int at, int range, int option)
```

Comment:
@highker suggested adding `Options.DEFAULT` directly here instead of passing it in since it is always the same.